homeassistant/components/bring/todo.py

```diff
@@ -215,3 +238,33 @@ async def async_delete_todo_items(self, uids: list[str]) -> None:
             raise HomeAssistantError("Unable to delete todo item for bring") from e
 
         await self.coordinator.async_refresh()
+
+    async def async_send_message(
+        self,
+        message: BringNotificationType,
+        item: str | None = None,
+    ) -> None:
+        """Send a push notification to members of a shared bring list."""
+
+        try:
+            await self.coordinator.bring.notify(self._list_uuid, message, item or None)
+        except BringRequestException as e:
+            raise HomeAssistantError(
+                translation_domain=DOMAIN,
+                translation_key="notify_request_failed",
+            ) from e
+        except ValueError as e:
```

Comment:
`ValueError` is very broad, are we sure it's only thrown when the item is missing? Maybe it would be better to check this ourselves?

Reply:
pretty sure, i wrote the notify method of the bring-api library.

Reply:
also rewrote the error message to be more consistent with other error messages.